Dockerfile

```diff
@@ -1,23 +1,24 @@
 ARG 	CHEMISTRY_PACKAGE=jchem
-FROM maven:3-openjdk-8 as builder
+FROM maven:3-openjdk-11 as builder
 
 FROM 	builder as dependencies
 ARG     CHEMISTRY_PACKAGE
 ENV     CHEMISTRY_PACKAGE=${CHEMISTRY_PACKAGE}
 
 FROM 	dependencies as jchem
-ADD 	lib/jchem-16.4.25.0.jar /lib/jchem-16.4.25.0.jar
-RUN     mvn install:install-file -Dfile=/lib/jchem-16.4.25.0.jar -DartifactId=jchem -DgroupId=com.chemaxon -Dversion=16.4.25.0 -Dpackaging=jar -DgeneratePom=true -DcreateChecksum=true
+# ADD 	lib/jchem-16.4.25.0.jar /lib/jchem-16.4.25.0.jar
```

Comment:
Previously we'd been assuming this jar file would be supplied despite it not being in the repo. Perhaps you could do something similar for the API keys?